src/main/java/com/exadel/discount/service/CountryService.java

```diff
@@ -0,0 +1,18 @@
+package com.exadel.discount.service;
+
+import com.exadel.discount.dto.CountryDTO;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface CountryService {
+    List<CountryDTO> findAll();
```

Comment:
please add an empty line between class and method declaration

Reply:
Done